packages/dai-plugin-mcd/scripts/prepublish.sh

```diff
@@ -4,6 +4,20 @@ if [ $(basename $(pwd)) != "dai-plugin-mcd" ]; then
   echo "This script must be run from the dai-plugin-mcd directory."
   exit
 fi
+
+yarn config set version-tag-prefix "dai-plugin-mcd-v"
+yarn config set version-git-message "dai-plugin-mcd-v%s"
+yarn version
+
+PACKAGE_VERSION=$(cat package.json \
+  | grep version \
+  | head -1 \
+  | awk -F: '{ print $2 }' \
+  | sed 's/[",]//g')
+
+git push origin dev --no-verify
+git push origin "dai-plugin-mcd-v${PACKAGE_VERSION:1}"
```

Comment:
This will run all the tests, which could be a good thing or it could be a nuisance. We can easily skip them (like on the line above) if we don't want to run them before publishing.

Reply:
if your intent here is just to push the tag, you can do `git push --tags`

Reply:
but actually i think we should not be pushing to git from the prepublish script. conceptually it feels like two different steps to me. and practically speaking, if we're building more than one package at once, then the multiple pushes are redundant. once all the version bump commits and tags are created, a single `git push --tags` at the end should take care of everything.

Reply:
Cool. I'll just communicate with the team that they'll need to push the extra commit and tag after publishing.